File: python/day2/Task2.py
def isScoreAndGrade(scoreOne, scoreTwo, scoreThree):
	average = (scoreOne + scoreTwo + scoreThree)// 3
	if average >= 90 and average < 100:
		return 'A'
	elif average >= 80 and average < 90:
		return 'B'
	elif average >= 70 and average < 80:
		return 'C'
	elif average >= 60 and average < 70:
		return 'D'
	else:
		return 'F'


print(isScoreAndGrade(10,20,90))

File: javascript/day2/Task2.js
function isScoreAndGrade(scoreOne,scoreTwo,scoreThree){
let average = (scoreOne + scoreTwo + scoreThree)/ 3;
if(average >= 90 && average <= 100)return 'A';
else if(average >= 80 && average < 90)return 'B';
else if(average >= 70 && average < 80)return 'C';
else if(average >= 60 && average < 70)return 'D';
else{
	return 'F';}
}

console.log(isScoreAndGrade(30,67,10));
